.github/workflows/cross_rs4s.yaml

```diff
@@ -19,7 +19,7 @@ jobs:
     runs-on: [ self-hosted, Linux, X64, cross-rs4s ]
     strategy:
       matrix:
-        core: ['3.1.2', '3.1.3', '3.1.4', '3.1.5', '3.2.0', '3.2.1', '3.2.2']
+        core: ['3.2.2', '3.1.2', '3.1.3', '3.1.4', '3.1.5', '3.2.0', '3.2.1'] # test last version first
```

Comment:
Good idea to start with the latest, but let's order them all as newest to oldest